### PR TITLE
Change in header above QR code

### PR DESCRIPTION
**Pull request by ShhubhamA**
1-Made a small change in header above QR code so that when the screenshot gets shared it communicates to the one who pays/send Clout!

2-Added the receiver  username :)

Send $CLOUT - BitClout - Google Chrome 08-07-2021 04_56_02 (2)

### CODE REVIEW

**Comment by harpreet9629** [comment]
That's great change

**Comment by ShhubhamA** [comment]
> That's great change

**Comment by ShhubhamA** [comment]
> That's great change

Idk lol maybe , maybe not and thanks for helping me run frontend locally :)

File: src/app/transfer-bitclout/transfer-bitclout.component.html [hunk 1]
```diff
@@ -90,7 +90,7 @@
 </button>
 
 <div class="p-15px d-flex flex-column align-items-center">
-  <p>Share to receive $CLOUT</p>
+  <p>Scan QR to Send {{ globalVars.loggedInUser.ProfileEntryResponse.Username }} $Clout</p>
```
Comment (maebeam) [inline src/app/transfer-bitclout/transfer-bitclout.component.html:93]: Can we do `Scan QR code to send {{ globalVars.loggedInUser.ProfileEntryResponse?.Username }} $CLOUT`?

Don't forget a user can be logged in and not have a `ProfileEntryResponse`

**Review by maebeam — CHANGES_REQUESTED** [review]

File: src/app/transfer-bitclout/transfer-bitclout.component.html (hunk 1 above)
Comment (ShhubhamA) [reply src/app/transfer-bitclout/transfer-bitclout.component.html]: Done :)

**Review by maebeam — APPROVED** [review]

**Review by lazynina — APPROVED** [review]

**Pull request merged**